arcgis-android-toolkit/src/main/res/values/integers.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Copyright 2019 Esri
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~    http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<resources>
+    <!-- ArCalibrationView -->
+    <item name="calibration_elevation_control_min" format="float" type="integer">-50.0</item>
+    <item name="calibration_elevation_control_max" format="float" type="integer">50.0</item>
+    <item name="calibration_heading_control_min" format="float" type="integer">-10.0</item>
+    <item name="calibration_heading_control_max" format="float" type="integer">10.0</item>
```

Comment:
Seems odd that these are "integers" when they have float values?

Reply:
Yeah, it's just the way we have to declare floats in XML. 

https://blog.danlew.net/2015/01/06/handling-android-resources-with-non-standard-formats/